File: javascript/practice_1.js
// PracticeA - Welcome to AtCoder

const input = require("fs").readFileSync("/dev/stdin", "utf8").split("\n");
const A = Number(input[0]);
const [B, C] = input[1].split(" ").map(Number);
const S = input[2];
console.log(A + B + C, S);

File: haskell/practice_1.hs
-- PracticeA - Welcome to AtCoder

main :: IO ()
main = do
  a <- readLn :: IO Int
  [b, c] <- map (read :: String -> Int) . words <$> getLine
  s <- getLine
  putStrLn $ unwords [show (a + b + c), s]
